lib/actions/reconcile-issue.js

```diff
@@ -0,0 +1,159 @@
+"use strict";
```

Comment:
This looks like transpiled output. Should it be in .gitignore?

Reply:
That might be nice. I wonder if it would be a good idea in the meantime to add a pre- or post-commit hook that automatically runs the build, so we don't forget to rebuild when committing TS changes? Probably overkill. Your solution sounds more robust.